config/lib/app.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var config = require('../config'),
+	mongoose = require('./mongoose'),
+	express = require('./express'),
+	chalk = require('chalk');
+
+// Initialize Models
+mongoose.loadModels();
```

Comment:
Probably. The point is that you can't really "test" or get anything to function without loading the models because they are part of the app. But there could be some cases where you want to load the app but not get it connected to the db or not load express (which is what init does) and cases where you also not want to start the app and make it listen (where is what start does).

Basically it could be argued that the loadModels is redundant because it is already called when the app.js is required. Thing is, I don't know yet in advance but there could be cases where you want to load them again.